core/src/main/web/app/helpers/helper.js

```diff
@@ -63,7 +63,112 @@
       newSession: function(empty) {
         return bkCoreManager.newSession(empty);
       },
-
```

Comment:
are all these changes electron specific?

can we put this logic in a more descriptive place as compared to just 'helper' ?